splice/queries/distribution.py

```diff
@@ -0,0 +1,134 @@
+import hashlib
+import os
+import urllib
+import json
+
+from datetime import datetime
+from collections import defaultdict
+from collections import namedtuple
+
+from sqlalchemy.sql.expression import false
+from splice.models import Tile, Adgroup, Campaign, CampaignCountry
+from splice.web.api.tile_upload import load_bucketer
+
+
+Dists = namedtuple("Dists", ['legacy', 'directory', 'suggested'])
+
+
+def _tile_dict(tile, bucketer, legacy=False):
+    tile_dict = dict(
+        directoryId=tile.id,
+        url=tile.target_url,
+        bgColor=tile.bg_color,
+        title=tile.title,
+        type=tile.type,
+        imageURI=switch_to_cdn_url(tile.image_uri),
+        enhancedImageURI=switch_to_cdn_url(tile.enhanced_image_uri))
+    if not legacy:
+        # only suggested tiles have categories
+        if tile.adgroup.categories:
+            # TODO(najiang@mozilla.com) only grab the first category here.
+            # We should create multiple tiles for multiple categories, with
+            # each tile has a frecent site list based on the category.
+            bucket = bucketer[tile.adgroup.categories[0].category]
```

Comment:
I read the comment above but still don't understand why you only use the first category.?